clients/java/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+}
+
+group 'signalr'
+version '0.1.0'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
```

Comment:
Looks like `testCompile` has been deprecated in favor of `testImplementation`: https://docs.gradle.org/current/userguide/java_plugin.html#tab:configurations